source/cloud/aws/sagemaker.md

```diff
@@ -41,7 +41,7 @@ set -e
 
 sudo -u ec2-user -i <<'EOF'
 
-mamba create -y -n rapids {{ rapids_conda_channels }} {{ rapids_sagemaker_conda_packages }} \
+mamba create -y -n rapids -c rapidsai -c conda-forge -c nvidia rapids=24.12 python=3.12 cuda-version=12.4 \
```

Comment:
Let's make a note to revert this once Sagemaker no longer has an upper limit on these values